apps/src/templates/sectionSetup/InitialSectionCreationInterstitial.jsx

```diff
@@ -0,0 +1,99 @@
+import React, {Component} from 'react';
+import PropTypes from 'prop-types';
+import PadAndCenter from '@cdo/apps/templates/teacherDashboard/PadAndCenter';
+import i18n from '@cdo/locale';
+import BaseDialog from '@cdo/apps/templates/BaseDialog';
+import Button from '@cdo/apps/templates/Button';
+import color from '@cdo/apps/util/color';
+import {connect} from 'react-redux';
+import {beginEditingSection} from '@cdo/apps/templates/teacherDashboard/teacherSectionsRedux';
+
+class InitialSectionCreationInterstitial extends Component {
+  static propTypes = {
+    onClose: PropTypes.func,
+    isOpen: PropTypes.bool,
+    // Connected to Redux
+    beginEditingSection: PropTypes.func.isRequired
+  };
+
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      isOpen: props.isOpen || true
```

Comment:
I'm not sure I get this- won't this always compute to true?

Reply:
This covers the case where a `props.isOpen` is not provided and in that case, defaults to `true`/open. However, I looked a little deeper here and realized I was saving the state and manually unmounting it - so I did a bit of adjusting here and ended up just defaulting this to set `isOpen: true`. This way, when we call the interstitial, it will be open by default until it is closed.

Reply:
Okay. Sounds good to me!